akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala

```diff
@@ -171,7 +178,7 @@ object Source extends SourceApply {
    * that mediate the flow of elements downstream and the propagation of
    * back-pressure upstream.
    */
-  def apply[T](publisher: Publisher[T]): Source[T, Unit] =
+  def apply[T](publisher: Publisher[_ >: T]): Source[T, Unit] =
```

Comment:
Here it is even more wrong, since Scala does have declaration-site variance, so this will never be useful (unless I am missing something).